crates/cext/src/transpiler/passes/vf2.rs

```diff
@@ -118,40 +120,180 @@ pub unsafe extern "C" fn qk_vf2_layout_result_free(layout: *mut VF2LayoutResult)
+/// the circuit.
 ///
-/// VF2Layout is a pass for choosing a layout of a circuit onto a connectivity graph as
-/// a subgraph isomorphism problem solved by VF2.
+/// This function corresponds to the Python-space ``VF2Layout`` pass.
+///
+/// This function is suitable for use on circuits that have not yet been fully lowered to hardware.
+/// If your circuit has already been completely lowered to hardware and you are looking to _improve_
+/// the layout for an exact interaction graph, use ``qk_transpile_pass_standalone_vf2_layout_exact``
+/// instead.
 ///
 /// If this pass finds a solution that means there is a "perfect layout" and that no
 /// further swap mapping or routing is needed. However, there is not always a possible
 /// solution, or a solution might exist but it is not found within the limits specified
 /// when the pass is called.
 ///
-/// By default, this pass will construct a heuristic scoring map based on the error rates
-/// in the provided ``target`` argument. The function will continue searching for layouts
-/// and use the heuristic scoring to return the layout which will run with the best estimated
-/// fidelity.
-///
 /// @param circuit A pointer to the circuit to run VF2Layout on
 /// @param target A pointer to the target to run the VF2Layout pass on
-/// @param strict_direction If true the pass will consider the edge direction in the
-///     connectivity described in the ``target``. Typically setting this to ``false``
-///     is desirable as an undirected search has more degrees of freedom and is more likely
-///     to find a layout (or a better layout if there are multiple choices) and correcting
-///     directionality is a simple operation for later transpilation stages.
-/// @param call_limit The number of state visits to attempt in each execution of the VF2 algorithm.
-///     If the value is set to a negative value the VF2 algorithm will run without any limit.
-/// @param time_limit The total time in seconds to run for ``VF2Layout``. This is checked after
-///     each layout search so it is not a hard time limit, but a soft limit that when checked
-///     if the set time has elapsed the function will return the best layout it has found so
-///     far. Set this to a value less than or equal to 0.0 to run without any time limit.
-/// @param max_trials The maximum number of trials to run the VF2 algorithm to try and find
-///     layouts. If the value is negative this will be treated as unbounded which means the
-///     algorithm will run until all possible layouts are scored. If the value is 0 the number
-///     of trials will be limited based on the number of edges in the interaction or the coupling
-///     graph (whichever is larger).
+/// @param config A pointer to the ``QkVF2LayoutConfiguration`` configuration structure.  If this
+///     pointer is null, the pass defaults are used.
+/// @param strict_direction If true, the pass will consider the edge direction in the
```

Comment:
```suggestion
/// @param strict_direction If ``true``, the pass will consider the edge direction in the
```

Reply:
I forgot this, but we can just apply it after #15229 to get the C API CI fix.